packages/patternfly-4/react-core/src/components/Select/GroupedCheckboxSelectInput.js

```diff
@@ -65,7 +65,7 @@ class GroupedCheckboxSelectInput extends React.Component {
           selections={selected}
           isExpanded={isExpanded}
           placeholderText="Filter by status"
-          ariaLabelledBy={titleId}
+          aria-labelledby={titleId}
```

Comment:
Same issue as previous comment... The old prop ariaLabelledBy is still being used in the example.